framework/src/org/checkerframework/framework/util/AnnotatedTypes.java

```diff
@@ -1425,13 +1425,14 @@ public static boolean isTypeAnnotation(AnnotationMirror anno) {
      *
      * @param elements
      *            an array of {@link ElementType} values
+     * @param cls the annotation class being tested; used for diagnostic messages only
      * @return true if the array only has {@link ElementType#TYPE_USE} (and
      *         optionally {@link ElementType#TYPE_PARAMETER}), false if it
```

Comment:
This method never returns false.  Instead it throws an exception via ErrorReport and an error is reported to the user.  Would you please document this?